runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java

```diff
@@ -95,12 +95,13 @@ protected void startUp() throws Exception
 		BufferedImage icon;
 		synchronized (ImageIO.class)
 		{
-			icon = ImageIO.read(getClass().getResourceAsStream("hiscore.gif"));
+			icon = ImageIO.read(getClass().getResourceAsStream("normal.png"));
```

Comment:
Why was this changed from hiscore.gif to normal.png?

Reply:
the straight-from-the-sprites hiscore.gif has a weird empty right side border, this makes it not horizontally centered.